test/DebugInfo/NonSemantic/static_member_array.ll

```diff
@@ -66,10 +66,10 @@ target triple = "spir64-unknown-unknown"
 !9 = !DIBasicType(name: "int", size: 32, encoding: DW_ATE_signed)
 !10 = !{!11}
 !11 = !DISubrange(count: 4)
-!12 = !DIDerivedType(tag: DW_TAG_member, name: "smem", scope: !13, file: !3, line: 4, baseType: !16, flags: DIFlagStaticMember)
+!12 = !DIDerivedType(tag: DW_TAG_variable, name: "smem", scope: !13, file: !3, line: 4, baseType: !16, flags: DIFlagStaticMember)
```

Comment:
This might not be a good fix here -- this testcase was generated and claimed to be DWARF 4 (See below !19 = !{i32 2, !"Dwarf Version", i32 4}). We shouldn't try to modify it to DWARF5.

I think we may need to create another DWARF5 test using new clang, while keep this one, as we will need to support DWARF 4 producer as wel.

Reply:
Thanks for noticing, you are right. We will fix it later, we just want to have less customizations in intel/llvm